packages/veritone-react-common/src/components/MediaInfoPanel/EditMetadataDialog/index.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
```

Comment:
all of the validation functions seem like unnecessary bloat, just do the comparison where you need the validation - no need to create a function for each one.

Reply:
Resolved this in PR 138